pyaedt/modeler/cad/Primitives.py

```diff
@@ -1310,8 +1309,8 @@ def get_obj_id(self, objname):
             Object ID.
 
         """
-        if objname in self.object_id_dict:
-            return self.object_id_dict[objname]
+        if objname in self._object_id_to_names:
+            return self._object_id_to_names[objname]
```

Comment:
Here we pass the name as argument and we get back the id. So, should the name be 'object_name_to_id'?
I mean, the dictionary goes from name to id. Am I wrong?